app/views/account/calculators/index.html.erb

```diff
@@ -25,7 +26,7 @@
     </thead>
     <tbody class="admin-table-links">
       <% @calculators.each do |calculator| %>
-        <tr class="<%= calculator.preferable ? "table-success" : "table-light" %>">
+        <tr>
```

Comment:
```suggestion
        <tr class="<%= calculator.preferable ? "table-success" : "text-inherit" %>">
```
So the preferable calculator is highlighted.

Reply:
Fixed.